scripts/setup

```diff
@@ -37,10 +37,10 @@ cds index queue init purge
 cds users create test@test.ch -a --password=123456
 # Create an admin user
 cds users create admin@test.ch -a --password=123456
-cds roles create admin
-cds roles add admin@test.ch admin
-cds access allow deposit-admin-access role admin
-cds access allow superuser-access role admin
+cds roles create cds-operators
```

Comment:
shall we revert this and below and move it to our deployment code?

Reply:
done